sklearn/metrics/pairwise.py

```diff
@@ -1049,6 +1049,11 @@ def distance_metrics():
     return PAIRWISE_DISTANCE_FUNCTIONS
 
 
+def _dist_wrapper(dist, mat, slice, *args, **kwargs):
```

Comment:
maybe use more explicit names?
```
def _dist_wrapper(dist_func, dist_matrix, col_slice, *args, **kwargs):
```

Reply:
ok. Keeping `slice` and not `col_slice` for now because of my answer to you comment below.